src/remoteConnector.ts

```diff
@@ -650,9 +651,7 @@ export class RemoteConnector extends Disposable {
 		this.usePublicApi = await this.experiments.getRaw<boolean>('gitpod_experimental_publicApi', { gitpodHost: params.gitpodHost }) ?? false;
 		this.logService.info(`Going to use ${this.usePublicApi ? 'public' : 'server'} API`);
 
-		const forceUseLocalApp = getServiceURL(params.gitpodHost) === 'https://gitpod.io'
-			? (await this.experiments.get<boolean>('gitpod.remote.useLocalApp', { gitpodHost: params.gitpodHost }))!
-			: (await this.experiments.get<boolean>('gitpod.remote.useLocalApp', { gitpodHost: params.gitpodHost }, 'gitpod_remote_useLocalApp_sh'))!;
```

Comment:
values were set to `false` for 100% users in configcat for a long time so let's remove this already to make things easier